interface/patient_file/encounter/forms.php

```diff
@@ -598,7 +629,7 @@ function divtoggle(spanid, divid) {
         }
 
         if (acl_check('admin', 'super') ) {
-            if ( $formdir != 'newpatient') {
+            if ( $formdir != 'newpatient' || $formdir != 'newGroupEncounter') {
```

Comment:
We need this for a new group encounter. It needs to be like the new patient encounter.